config.go

```diff
@@ -31,6 +31,9 @@ type Config struct {
 
 	// Response headers to add to every static.
 	Response map[string]string `mapstructure:"response"`
+
+	// GzipEnabled determines if gzip compression is enabled for serving static files.
+	GzipEnabled bool `mapstructure:"gzip_enabled"`
```

Comment:
I'm not sure this is a good name for the option. From my POV option name should be self-explanatory.
I think that we may remove that option at all and check the `gzip` header only.